mdx-models/src/main/java/com/mx/path/model/mdx/model/payment/Enrollment.java

```diff
@@ -0,0 +1,33 @@
+package com.mx.path.model.mdx.model.payment;
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+import com.mx.path.model.mdx.model.MdxBase;
+import com.mx.path.model.mdx.model.id.MfaChallenge;
+
+public final class Enrollment extends MdxBase<Enrollment> {
+  @SerializedName("is_active")
+  private Boolean isActive;
+
+  private List<MfaChallenge> challenges;
+
+  public Enrollment() {
+  }
+
+  public Boolean getActive() {
+    return isActive;
+  }
+
+  public void setActive(Boolean active) {
```

Comment:
setIsActive would be better since 'setActive' could be misread as 'set the Enrollment to be active' instead of "set the IsActive boolean's value"

Reply:
Done